command-line-interface.md

```diff
@@ -0,0 +1,246 @@
+# Operations
+
+The [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
Say you have a specification that requires implementations to support at least one of HTTP/1.1 or HTTP/2.  You use language like we have in our README:

> An implementation is not compliant if it fails to satisfy one or more of the MUST, REQUIRED, or SHALL requirements for the protocols it implements.  An implementation is compliant if it satisfies all the MUST, REQUIRED, and SHALL requirements for the protocols it implements.
> 
> Protocols defined by this specification are:
>
> * HTTP/1.1, as defined by RFC 2068.
> * HTTP/2.0, as defined by RFC 7540.

[HTTP/2 has][1]:

> Once TLS negotiation is complete, both the client and the server MUST send a connection preface (Section 3.5).

The [HTTP/1.1 spec][2], on the other hand, has no such handshake after TLS negotiation.  Is an implementation that implements valid HTTP1.1 noncompliant because it fails this unqualified HTTP/2 condition?  I don't think so.  It just implements the HTTP/1.1 protocol but not the HTTP/2 protocol.  You badge it as HTTP/1.1 compliant and mention that it's HTTP/2 noncompliant.

Similarly, I think an implementation that implements the future FooBar API (which a later PR lands in this spec) can be compliant Linux container (say) implementation even if it fails to implement the command line API.  You badge it as “OCI Linux container runtime on amd64 with the FooBar API” and mention that it's noncompliant with the command line API.  Is there anything wrong with that?

[1]: https://tools.ietf.org/html/rfc7540#section-3.3
[2]: https://tools.ietf.org/html/rfc2068